fedimint/src/consensus/mod.rs

```diff
@@ -106,6 +106,11 @@ where
         &self,
         transaction: Transaction,
     ) -> Result<(), TransactionSubmissionError> {
+        // we already processed the transaction before the request was received
```

Comment:
Discovered this bug when an epoch is processed before the submit_transaction occurs.